particles/Tutorial/Javascript/4_Handles/Handles.arcs

```diff
@@ -1,31 +1,30 @@
-// Tutorial: Handles
-// Shows how to use a handle to pass information between particles.
 
+// Define a schema that allows us to store a person's name 
 schema Person
   Text name
 
 // The GetPerson particle allows the user to input their name, then writes
 // the input to the Person handle.
 // This particle also provides a slot to display a greeting to the person.
 particle GetPerson in './source/GetPerson.js'
-  out Person person
+  out Person personOut
```

Comment:
Hmmm.... do we want to tech folks to put direction in the connection name?

Also, soon `out` will become `writes`. What do we do then?

Reply:
I really dislike the "people = people" syntax. I think it is confusing and doesn't help when you're first learning arcs as the difference between person and person is unclear. Having said that, I really dislike the "personOut" and "personIn" naming, so I changed them back to "person" and updated it on the recipe side to "recipePerson".  Is that a better solution?